docs/Deep-Dives/metaapi.md

```diff
@@ -0,0 +1,200 @@
+
+This is a crucial step in Flotiq. 
+Let's open your `API Doc` and download `OpenAPI Specification`.
+
+![Link to your personal API Doc](images/metaapi/meta-flotiq-3.png){: .border}
+
+![Download OpenAPI Specification button](images/metaapi/meta-flotiq-4.png){: .border}
+
+We use the content of the downloaded file in the next step.
+
+
+### 4. Add your Flotiq API to Meta-API
+
+!!! Note "Why there is no predefined Flotiq API in Meta-API catalog?"
+    The Main concept of Flotiq is that your API Specification literally represents your set of content definitions.
+    Your API represents your endpoints, with your models consisting of your properties.
+    In short, **Flotiq API contains different endpoints for each user to be more personalized**.
+
+We assume that you have an account on Meta-API platform and you are logged in. 
+Go to the Catalog of API and click "Add an API".
+
+![Add Flotiq API to Meta-API catalog](images/metaapi/meta-1.png){: .border}
+
+In the `Add an API to the catalog` modal, go to the bottom and fill the `Open API Specification` field with data
+retrieved in step 2. Other fields (except Category) should auto-complete.
+
+![Add an API dialog is the place where you paste OpenAPI Specification](images/metaapi/meta-2.png){: .border}
+
+And that's it. We have our private Flotiq API in our private Meta-API Catalog. 
+Now we can use it to create a `Spell`.
+
+
+### 5. Create a Spell that connects Flotiq with any other API
+
+When our API is added to Catalog, we can add a data flow between Flotiq and other applications. 
+Go to the Spells section in Meta-API dashboard and click `Create a new Spell`.
+
+![Crete a new Spell in Meta-API](images/metaapi/meta-3.png){: .border}
+
+In the `Choose the API` step, find your Flotiq API.
+
+![Find Flotiq API in the APIs list](images/metaapi/meta-4.png){: .border}
+
+Next, choose `Connector`, which means the configurable endpoint you want to call.
+Select a products list endpoint (`GET /api/v1/content/product`). Click `Use`.
+
+![Choose a products list endpoint from Flotiq API](images/metaapi/meta-5.png){: .border}
+
+Now it is time to connect Flotiq with other applications.
+
+In our example we choose `Google Spreadsheet API`, and `append` Connector 
+(which match `POST /v4/spreadsheets/{spreadsheetId}/values/{range}:append` endpoint).
+
+A "Google Spreadsheet Append" connector is responsible for appending the values to a selected spreadsheet.
+
+![Add Google Spreadsheet API to Spell](images/metaapi/meta-6.png){: .border}
+
+![Choose the "append" endpoint (Connector)](images/metaapi/meta-7.png){: .border}
+
+Next, click "Continue with 2 connectors" and go through the following steps with values to create your Spell.
+
+![Submit your spell](images/metaapi/meta-8.png){: .border}
+
+Now you should see the edit Spell view.
+
+![Edit spell view](images/metaapi/meta-9.png){: .border}
+
+
+## 6. Authenticate your Connectors
+
+As you can see in the right sidebar, there are a few alerts. 
```

Comment:
I think we should state what we are trying to achieve.
Add flotiq key, and how to do it ....
Add google key and how to ...
I think it is necessary to explain and not only drag the user through the interface.